### ISSUE TITLE
[Change] Sort by Most Recent Update in Recordings Screen

### ISSUE THREAD

**Issue by jasoncross** [syd711/vpin-studio#635]
It occurred to me today that it would be very useful to be able to sort the list of tables on the recording screen by last update.  That way I can see pretty quickly which tables have been updated recently and may need re-recording.

**Comment by syd711** [comment]
Good point. It's a bit tricky to tell which modification date to use (e.g. playfield vs. backglass). But I'll take a look.

**Comment by jasoncross** [comment]
I was just thinking whatever drives the "Last Updated" field on the main screen list. I can then just select all of my recently updated tables and do a re-record of all screens as that is easy enough.

**Comment by syd711** [comment]
I've added the "Last Updated" column at the end of the recorder table.

**Comment by syd711** [comment]
Fixed with 3.11.0

**Comment by jasoncross** [comment]
There seems to be a bug with this one.  The column shows but when you click it to sort it doesn't sort (despite showing the up / down arrows after being clicked).

**Comment by syd711** [comment]
Damn, yes, I forgot the sorter there. Comes with the next patch.

**Comment by syd711** [comment]
Fixed with 3.12.0
